### ISSUE TITLE
A11y: Paypal input fields (credit card) need labels

### ISSUE THREAD

**Issue by shopwareBot** [shopware/platform#5025]
Credit card input fields are missing labels
[https://www.w3.org/WAI/tutorials/forms/labels/|https://www.w3.org/WAI/tutorials/forms/labels/|smart-link] 

!image-20241008-112428.png|width=771,height=405,alt="image-20241008-112428.png"!

**Comment by shopwareBot** [comment]
[public] Shopware automated response: This issue is linked to the internal issue: **PPI-998**



[created from PPI-998, comment 501656]

**Comment by shopwareBot** [comment]
[public] 

 > Priority changed to: **Low**

 Thank you for your report. After careful consideration, we've concluded that this issue does not require immediate attention or isn't something we can address right now. We prioritize openness and clear communication with our community, so we want to be transparent. We may not resolve all reported issues, and this one might remain unresolved. To stay effective, we periodically clean our backlog, and this issue could be closed in the future.

 For now, we'll keep this ticket open to invite further feedback from the community in case we've underestimated its impact and to allow us to revisit it later.



[created from PPI-998, comment 539369]
